README.md

```diff
@@ -1 +1,180 @@
-# Starcounter.Linq
\ No newline at end of file
+# Starcounter.Linq
+
+Starcounter.Linq is a LINQ to SQL Provider for Starcounter Database. It uses standard LINQ syntax for queries.
+
+It is available for downloading as [Starcounter.Linq](https://www.nuget.org/packages/Starcounter.Linq/) NuGet package.
+
+## Requirements
+
+Requires Starcounter version 2.3.1 and .NET Framework 4.5.

```

Comment:
version 2.3.1 -> 2.3.1 or later

This should be added if it works with later versions